src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DefaultRazorDocumentMappingService.cs

```diff
@@ -334,6 +335,18 @@ public override bool TryMapToProjectedDocumentPosition(RazorCodeDocument codeDoc
         public override bool TryMapToProjectedDocumentOrNextCSharpPosition(RazorCodeDocument codeDocument, int absoluteIndex, [NotNullWhen(true)] out Position? projectedPosition, out int projectedIndex)
             => TryMapToProjectedDocumentPositionInternal(codeDocument, absoluteIndex, nextCSharpPositionOnFailure: true, out projectedPosition, out projectedIndex);
 
+        public override bool TryMapToVSProjectedDocumentPosition(RazorCodeDocument codeDocument, int absoluteIndex, [NotNullWhen(true)] out VSPosition? projectedPosition, out int projectedIndex)
```

Comment:
@davidwengier made a `RazorDocumentMappingService` Extension class, this is probably more appropriate there. (Also I don't remember the exact ones I created but someone is probably gonna have conflicts lol).

Reply:
ya good point 😄, once yours merges I'll rebase